runs/graphql/src/models/presentationdetails.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  FirebaseResult,
+  FirebaseResultItem,
+  FirebaseDocumentRef,
+  db
+} from "../services/firebase";
+import admin from "firebase-admin";
+
+export type PresentationDetails = {
+  timestamp?: string;
```

Comment:
Could this be a number?

Reply:
A good observation, in theory it should. The reason is that when I return it in graphql only 32 bit integers are allowed, and js timestamps are milliseconds so more than 32 bits. So some conversation is needed back and forth. 

But I will change it, it will allow a better sort in firebase queries 👍